### ISSUE TITLE
Vim swapfile issues: E72: Close error on swap or E200: *ReadPre autocommands made the file unreadable

### ISSUE THREAD

**Issue by TiBeN** [dsoprea/GDriveFS#194]
I use my Google Drive mainly for storing notes *.md (Markdown) files. I have some issues editing theses files with Vim. With no `~/.vimrc` file (no custom configuration), the file takes a lot of time to be opened with vim, then triggers the following error on close:

    E72: Close error on swap

I think this is because Vim is not able to create the swap file. Now if i put `set noswapfile` into a `~/.vimrc` file to disable swap file creation, Vim triggers the following error on file opening, leaving the editor with a blank page (file is not opened at all): 

    E200: *ReadPre autocommands made the file unreadable

I'm using Vim 7.4 on Ubuntu 16.04.

**Comment by cyanpencil** [comment]
I currently have the same error on Vim 8.1 with archlinux. Did you find a solution eventually?

**Comment by dsoprea** [comment]
I had actually originally tested the implementation, specifically, on VIM. There were issues with all of the opening and closing of so many different files all at once when first opening a single file, but they were cleared up. I'll take a look at this after clearing up some other issues.

It's not good etiquette to market for alternative projects from the issues of another.

**Comment by TiBeN** [comment]
@dsoprea, 
I'm sorry, my purpose was not to market anything but just answer to @cyanpencil question about the solution that worked for me.
I have true respect for the work you do on this project and will probably take another look at it when this issue will be solved.

**Comment by cyanpencil** [comment]
> It's not good etiquette to market for alternative projects from the issues of another.

Yes, in this case it actually is. He pointed out an open-source alternative that doesn't have this specific issue.  This is not marketing, nor him nor the author of `google-drive-ocamlfuse` gain anything by having one more user, as it's an open source product: that's why it's called an *alternative*, not a *competitor*. It's just that `google-drive-ocamlfuse` fits better one particular use case (editing files with vim) ; I'm sure there are a lot of other specific use cases where your product is better.
If anything, you should be thankful to @TiBeN: he opened this issue, pointing out a problem you weren't aware of; he also linked to a solution, as nothing stops you to go look in that project source code and see how they managed to avoid this issue. 

@TiBeN Thank you for opening this issue. Thank you also for suggesting an alternative that doesn't have this issue.
